x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -101,3 +103,13 @@ export const dragAndDropIdToggleFieldToTimeline = () => {
     drop(headersDropArea)
   );
 };
+
+export const removeColumn = (column: number) => {
+  cy.get(REMOVE_COLUMN).first().should('exist');
+  cy.get(REMOVE_COLUMN).eq(column).click({ force: true });
+  cy.wait(3000); // wait for DOM updates
```

Comment:
could we use some `cy.get` instead of `wait` to be more reliable?

Reply:
I agree with Patryk, using a hardcoded wait usually is not a good practice, this is because we can make the tests even slower if the action we are waiting to be performed is done before the hardcoded wait finishes.

I saw for events table that each time a column is deleted, a spinner is displayed in the timestamp column. What we can do is after clicking on the remove icon, wait first for the spinner to be present in the DOM and then wait for the spinner to disappear.

My proposal is to create a different method and call it after ```removeColumn```

Something like this:
```export const waitForTimelineColumnToBeRemoved = () => {
  cy.get(TIMELINE_COLUMN_SPINNER).should('exist');
  cy.get(TIMELINE_COLUMN_SPINNER).should('not.exist');
};```

Feel free to use a different naming, is just an example :)

Reply:
Thank you so much! I will do it this way.